tests/acceptance/features/apiSpaces/shareOperations.feature

```diff
@@ -224,4 +224,167 @@ Feature: sharing
+    And user "Alice" has created folder "FOLDER"
+    And user "Alice" has created a share with settings
+      | path        | FOLDER |
+      | shareType   | group  |
+      | permissions | create |
+      | shareWith   | grp1   |
+    And user "Brian" has accepted share "/FOLDER" offered by user "Alice"
+    When user "Brian" uploads a file inside space "Shares Jail" with content "new description" to "/FOLDER/textfile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And the following headers should match these regular expressions for user "Brian"
+      | ETag | /^"[a-f0-9:\.]{1,32}"$/ |
+    And the content of file "/FOLDER/textfile.txt" for user "Alice" should be:
+    """
+    new description
+    """
+
+
+  Scenario:  Uploading a file to a folder received as a read/write user share
+    Given user "Alice" has created folder "FOLDER"
+    And user "Alice" has created a share with settings
+      | path        | FOLDER |
+      | shareType   | user   |
+      | permissions | change |
+      | shareWith   | Brian  |
+    And user "Brian" has accepted share "/FOLDER" offered by user "Alice"
+    When user "Brian" uploads a file inside space "Shares Jail" with content "new description" to "/FOLDER/textfile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And the content of file "/FOLDER/textfile.txt" for user "Alice" should be:
+    """
+    new description
+    """
+
+
+  Scenario: Uploading a file to a folder received as a read/write group share
+    Given group "grp1" has been created
+    And user "Brian" has been added to group "grp1"
+    And user "Alice" has created folder "FOLDER"
+    And user "Alice" has created a share with settings
+      | path        | FOLDER |
+      | shareType   | group  |
+      | permissions | change |
+      | shareWith   | grp1   |
+    And user "Brian" has accepted share "/FOLDER" offered by user "Alice"
+    When user "Brian" uploads a file inside space "Shares Jail" with content "new description" to "/FOLDER/textfile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And the content of file "/FOLDER/textfile.txt" for user "Alice" should be:
+    """
+    new description
+    """
+
+
+  Scenario: Check quota of owners parent directory of a shared file
+    Given user "Alice" has uploaded file "filesForUpload/lorem.txt" to "/myfile.txt"
+    And user "Alice" has shared file "myfile.txt" with user "Brian"
+    And user "Brian" has accepted share "/myfile.txt" offered by user "Alice"
+    When user "Brian" uploads a file inside space "Shares Jail" with content "new description" to "/myfile.txt" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And the following headers should match these regular expressions for user "Brian"
+      | ETag | /^"[a-f0-9:\.]{1,32}"$/ |
+    And the content of file "/myfile.txt" for user "Alice" should be:
+    """
+    new description
+    """
```

Comment:
I don't see what this test has to do with quota

Reply:
I removed the scenario. Don't think it'll be relevant here